### PR TITLE
test: added refreshAccessToken integration tests

### PR DESCRIPTION
**Pull request by bhudnell**
Issue #, if available:

Description of changes:
- Added integration tests for the authentication package's `refreshAccessToken` API

Checklist:

* [x] Have you successfully deployed to an AWS account with your changes?
* [x] Have you written new tests for your core changes, as applicable?

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

File: workbench-core/example/infrastructure/integration-tests/tests/authentication/routeHandler.test.ts [hunk 1]
```diff
@@ -104,4 +107,29 @@ describe('authentication route handler integration tests', () => {
       expect(data.loggedIn).toBe(false);
     });
   });
+
+  describe('refreshAccessToken', () => {
+    it('should return the id token if the access token is successfully refreshed', async () => {
```
Comment (DaveMadsen) [inline workbench-core/example/infrastructure/integration-tests/tests/authentication/routeHandler.test.ts:112]: Do we need a test for an expired access token?

**Review by DaveMadsen — APPROVED** [review]
one comment, not blocking this code. Can be addressed here or another ticket if it needs to be addressed.

File: workbench-core/example/infrastructure/integration-tests/tests/authentication/routeHandler.test.ts (hunk 1 above)
Comment (bhudnell) [reply workbench-core/example/infrastructure/integration-tests/tests/authentication/routeHandler.test.ts:112]: In theory we probably should. In practice, the refresh token is stored in a cookie that expires and deletes at the same time the token itself expires so this is a very edge case. If we want to test it, it will take a decent amount of effort to figure out how to generate a valid but expired refresh token to test with. So I would say handle it in another ticket.
